README.md

```diff
@@ -0,0 +1,126 @@
+
+# What is the application?
+
+A simple REST API web service consisting of 2 microservices. 
+The application consists of a Java web application that exposes a REST API which talks to an Express application.
+
+### Endpoints
+
+#### Node Endpoints
+ 
+- **GET Hello World**
+
+    - **Request:**
+      - Method: GET
+      - URL: `/greeting`
+      - Example Request:
+        ```
+        GET /greting
+        ```
+
+    - **Response:**
+        - Status Code: 200 OK
+        - Example Response:
+          ```
+          { 
+           "text": "Hello World"
+          }
+          ```
+
+
+   - **Send Greeting**
+   - **Request:**
+      - Method: POST
+      - URL: `/process-greeting-node `
+      - Example Request:
+      ```
+       POST /nodeendpoint 
+       Content-Type: application/json
+    
+              {
+                "name": "Hello Ali"
+              }
+     ```
+   - **Response:**
+       - Status Code: 200 OK
+       - Example Response:
+       ```
+         {  
+          "name": "Hello from Ali" 
+         }
+       ```
+
+#### Dropwizard Endpoints
+
+- **A GET HELLOWorld.**
+
+    - **Request:**
+        - Method: GET
+        - URL: `greeting`
+        - Example Request:
+          ```
+          GET /greeting
+          ```
+
+    - **Response:**
+        - Status Code: 200 OK
+        - Example Response:
+          ```
+          {
+          "text": "hello world"
+          }
+          ```
+
+
+- **Send Greeting**
+- **Request:**
+    - Method: POST
+    - URL: `/process-greeting `
+    - Example Request:
+   ```
+    POST /process-greeting
+    Content-Type: application/json
+ 
+           {
+             "name": "Hello from John"
+           }
+  ```
+  - **Response:**
+      - Status Code: 200 OK
+      - Example Response:
+      ```
+       { 
+         "name": "Hello from John" 
+       }
+      ```
+
+
+### How the application works
+
+- The `HelloWorldResource` handles HTTP requests and forwards them to the appropriate endpoints in the Node.js service using
+  JAX-RS as a rest client
+- Responses from the Node.js service are then processed by the `HelloWorldResource`  and sent back to the client. 
+- Jackson is used for deserialising `app.yml` into AppConfiguration.
+- Jackson is also used for deserialising the requests/responses to/from Dropwizard 
+
+
+### How the test works ####
+ - The integration test uses Wiremock and rest Assured. It uses WireMock to stub out requests to the Node application.
+ - The test spins up a WireMock instance with a port number obtained from config/test-it.yml. 
+ This file is deserialised to `AppConfiguration` and so effectively overrides the original `app.yml` pointing Dropwizard to the Wiremock instance.
```

Comment:
Is there meant to be a bullet point here? Also make sure file names use single ticks.